recipes/libnl/libnl_3.2.28.oe

```diff
@@ -0,0 +1,28 @@
+require ${PN}.inc
+
+LICENSE = "LGPLv2.1+"
```

Comment:
could you make a "fix license" commit, where this is fixed to use spdx: LGPL-2.1+